From 9b2b28a314e56f6334b99de9e5345811a6385ca4 Mon Sep 17 00:00:00 2001
From: yixiangzhike <yixiangzhike007@163.com>
Date: Thu, 13 Jan 2022 10:58:52 +0800
Subject: [PATCH]  revert "remove authselect compat package"

---
 Makefile.am | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 9e1ea66..5ceeaca 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,6 +2,7 @@ SUBDIRS= \
     po \
     profiles \
     src/common \
+    src/compat \
     src/lib \
     src/cli \
     src/tests \
@@ -23,22 +24,8 @@ EXTRA_DIST = \
     scripts/release-version.sh \
     scripts/po-linguas.sh \
     scripts/zanata-hooks.sh \
-    src/compat/authcompat_ConfigSnippet.py \
-    src/compat/authcompat_EnvironmentFile.py \
-    src/compat/authcompat_Options.py \
-    src/compat/authcompat.py.in.in \
-    src/compat/Makefile.am \
-	src/compat/Makefile.in \
-    src/compat/snippets \
-    src/compat/snippets/authconfig-krb.conf \
-    src/compat/snippets/authconfig-sssd.conf \
     $(NULL)
 
-CLEANFILES = \
-    src/compat/authcompat.py.in \
-    src/compat/Makefile \
-	$(NULL)
-
 # Create authselect directories
 authselect_config_dir=@AUTHSELECT_CONFIG_DIR@
 authselect_vendor_dir=@AUTHSELECT_VENDOR_DIR@
-- 
1.8.3.1

